### PR TITLE
Update soroban's help output.

### PR DESCRIPTION
**Pull request by fnando**

### CODE REVIEW

File: docs/smart-contracts/getting-started/setup.mdx [hunk 1]
```diff
@@ -84,7 +84,7 @@ The [Soroban CLI](http://github.com/stellar/soroban-cli) can execute Soroban con
 Install the [latest released version](https://github.com/stellar/soroban-cli/releases) of Soroban CLI using `cargo install`.
 
 ```sh
-cargo install --locked soroban-cli --version 21.0.0-rc.1
```
Comment (willemneal) [inline docs/smart-contracts/getting-started/setup.mdx:87]: Aren't we moving toward `stellar-cli`?

File: docs/smart-contracts/getting-started/setup.mdx [hunk 2]
```diff
@@ -105,29 +105,28 @@ soroban
...
 
 Usage: soroban [OPTIONS] <COMMAND>
 
 Commands:
   completion  Print shell completion code for the specified shell
+  config      Deprecated, use `soroban keys` and `soroban network` instead
   contract    Tools for smart contract developers
   events      Watch the network for contract events
   keys        Create and manage identities including keys and addresses
-  xdr         Decode and encode XDR
```
Comment (willemneal) [inline docs/smart-contracts/getting-started/setup.mdx:118]: This wasn't removed.

File: docs/smart-contracts/getting-started/setup.mdx [hunk 3]
```diff
@@ -105,29 +105,28 @@ soroban
...
+  lab         Experiment with early features and expert tools
   network     Start and configure networks
   version     Print version information
-  cache       Cache for tranasctions and contract specs
```
Comment (willemneal) [inline docs/smart-contracts/getting-started/setup.mdx:121]: This was added recently.

File: docs/smart-contracts/getting-started/setup.mdx (hunk 1 above)
Comment (fnando) [reply docs/smart-contracts/getting-started/setup.mdx:87]: yes, there's a separate pr for that. #633

File: docs/smart-contracts/getting-started/setup.mdx (hunk 2 above)
Comment (fnando) [reply docs/smart-contracts/getting-started/setup.mdx:118]: this is exactly what the cli is showing (`soroban`). Can you please confirm the output on your end?

File: docs/smart-contracts/getting-started/setup.mdx (hunk 3 above)
Comment (fnando) [reply docs/smart-contracts/getting-started/setup.mdx:121]: like my previous comment, this is exactly what the released cli is returning right now.

File: docs/smart-contracts/getting-started/setup.mdx (hunk 1 above)
Comment (janewang) [reply docs/smart-contracts/getting-started/setup.mdx:87]: I can merge #633 but there will be multiple PRs to move all the docs over.

Comment (fnando) [reply docs/smart-contracts/getting-started/setup.mdx:87]: I think it's fine if we go over #633 slowly. I noticed there are a few conflicts to resolve.

Comment (janewang) [reply docs/smart-contracts/getting-started/setup.mdx:87]: I think it's going to take more than a few pass. I'd refer to keep the commits small so there are less merge conflicts.

File: docs/smart-contracts/getting-started/setup.mdx [hunk 4]
```diff
@@ -95,18 +95,33 @@ Report issues and share feedback about the Stellar CLI [here](https://github.com
...
+Run the `stellar --help` command and you should see output like below.
 
 ```console
-$ stellar
+$ stellar --help
 Build, deploy, & interact with contracts; set identities to sign with; configure networks; generate keys; and more.
 
 Stellar Docs: https://developers.stellar.org
-CLI Reference: https://github.com/stellar/stellar-cli/blob/main/FULL_HELP_DOCS.md
+CLI Full Hep Docs: https://github.com/stellar/stellar-cli/tree/main/FULL_HELP_DOCS.md
```
Comment (willemneal) [inline docs/smart-contracts/getting-started/setup.mdx:105]:
```suggestion
CLI Full Help Docs: https://github.com/stellar/stellar-cli/tree/main/FULL_HELP_DOCS.md
```

File: docs/smart-contracts/getting-started/setup.mdx [hunk 5]
```diff
@@ -95,18 +95,33 @@ Report issues and share feedback about the Stellar CLI [here](https://github.com
...
+
+The easiest way to get started is to generate a new identity:
+
+    stellar config identity generate alice
```
Comment (willemneal) [inline docs/smart-contracts/getting-started/setup.mdx:109]:
```suggestion
    stellar keys generate alice
```

**Review by willemneal — APPROVED** [review]
Just a couple typos

File: docs/smart-contracts/getting-started/setup.mdx (hunk 4 above)
Comment (fnando) [reply docs/smart-contracts/getting-started/setup.mdx:105]: oh, we need to fix this on the cli then.

File: docs/smart-contracts/getting-started/setup.mdx (hunk 5 above)
Comment (fnando) [reply docs/smart-contracts/getting-started/setup.mdx]: fix here and also on stellar/stellar-cli.

File: docs/smart-contracts/getting-started/setup.mdx (hunk 4 above)
Comment (fnando) [reply docs/smart-contracts/getting-started/setup.mdx]: fixed on stellar/stellar-cli as well.

**Review by ElliotFriend — APPROVED** [review]
Thanks for updating this!!

**Pull request merged**